.github/workflows/debian.yml

```diff
@@ -48,6 +48,41 @@ jobs:
             lib: "-stdlib=libc++",
             testRegex: "cpp"
            }
+          - {
+            name: "Clang-16-Python",
+            config: Release,
+            compilerC: clang-16,
+            compilerCxx: clang++-16,
+            lib: "-stdlib=libc++",
+            testRegex: "python"
+           }
+    steps:
+      - uses: actions/checkout@v4
+        with:
+         path: 'repo'
+      - name: Build
+        working-directory: ./repo
+        run: |
+          mkdir cmake-build-${{ matrix.config.config }}
+          cd cmake-build-${{ matrix.config.config }}
+          cmake ../  -DCMAKE_BUILD_TYPE=${{ matrix.config.config }} -GNinja -DCMAKE_C_COMPILER=${{ matrix.config.compilerC }} -DCMAKE_CXX_COMPILER=${{ matrix.config.compilerCxx }}
+          cmake --build . --parallel 2 --target ikarus
+          cmake --build . --parallel 2 --target _ikarus
```

Comment:
Here you also have to build `_io` maybe? Then maybe it should really be a submodule because otherwise it will not work directly?